java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -47,6 +59,16 @@ public interface StringCommands {
      *     {@link ConditionalSet#ONLY_IF_EXISTS} or {@link ConditionalSet#ONLY_IF_DOES_NOT_EXIST}
      *     conditions, return <code>null</code>. If {@link SetOptionsBuilder#returnOldValue(boolean)}
      *     is set, return the old value as a <code>String</code>.
+     * @example
+     *     <pre>
+     * String payload =
+     *         client.set("key", "value", SetOptions.builder()
+     *                 .conditionalSet(ONLY_IF_EXISTS)
```

Comment:
what's this indentation? I don't understand what the lined is aligning with.

Reply:
Spotless seems to be causing it.